src/core/net_db.cc

```diff
@@ -104,7 +104,7 @@ void NetDb::Run() {
     try {
       auto msg =
 	m_Queue.GetNextWithTimeout(
-	     static_cast<std::size_t>(NetDbDuration::WaitForMessageTimeout));  // 15 sec
+            static_cast<std::size_t>(NetDbDuration::WaitForMessageTimeout));  // 15 sec
```

Comment:
If you're going to newline ident for assignments, I'd advise something like this:

```cpp
auto msg =
  m_Queue.GetNextWithTimeout(  // 2 spaces
      static_cast<std::size_t>(NetDbDuration::WaitForMessageTimeout));  // 4 spaces for function parameters
```